src/Authentication/appsettings.json

```diff
@@ -29,5 +29,14 @@
     "CertificatePath": "jwtselfsignedcert.pfx"
   },
   "OidcProviders": {
+  },
+  "QueueStorageSettings": {
+    "ConnectionString": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1",
+    "QueueEndpoint": "http://127.0.0.1:10001/devstoreaccount1",
+    "AccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
+    "AuthenticationEventQueueName": "eventlog"
+  },
+  "FeatureManagement": {
+    "AuditLog": true
```

Comment:
Should this be true by default in appsettings.json? or set pr. environment through values.yaml?

Reply:
Good question. I think i will set this to false by default and turn on in a specific environment where i can test it out